Assets/Scripts/InventoryManager.cs

```diff
@@ -58,27 +59,20 @@ private void Awake()
     // Sets inventory to be closed when the game starts
     void Start()
     {
-        inventoryGUI.SetActive(false);
         HeldObject = null;
-        InputSystem.actions.FindAction("ToggleInventory").performed += ctx => ToggleInventoryGUI();
         SwitchTab(0);
     }
     
     // This toggles the state of the Inventory GUI (open or closed)
-    private void ToggleInventoryGUI()
+    public void SetInventoryActiveState(bool isActive)
```

Comment:
Nice! This follows the standards for naming and the parameter is easy to understand when trying to use the method!